### ISSUE TITLE
Pre-trained ResNet50 model on ImageNet?

### ISSUE THREAD

**Issue by wangzheallen** [miguelvr/dropblock#38]
Do you have pre-trained model released?
Or we just change the dropout to dropblock in  pre-trained ResNet 50 is ok?

**Comment by miguelvr** [comment]
no, I don't have any pre-trained models available.

you would have to add dropblock to your network an re-train.
